src/server.c

```diff
@@ -4839,16 +4838,29 @@ sds genRedisInfoString(const char *section) {
     /* CPU */
     if (allsections || defsections || !strcasecmp(section,"cpu")) {
         if (sections++) info = sdscat(info,"\r\n");
-        info = sdscatprintf(info,
+
+        struct rusage self_ru, c_ru;
+        getrusage(RUSAGE_SELF, &self_ru);
+        getrusage(RUSAGE_CHILDREN, &c_ru);
+        info = sdscatfmt(info,
```

Comment:
reverting your change.
```suggestion
        info = sdscatprintf(info,
```